docs/config.rst

```diff
@@ -63,6 +63,13 @@ A list of configuration keys currently understood by the extension:
                                    that it will be disabled by default in
                                    the future.  This requires extra memory
                                    and should be disabled if not needed.
+``SQLALCHEMY_ISOLATE_TRANSACTION`` Controls the default behavior of root
```

Comment:
Why did you remove only one of the options?

Reply:
Because I understood it like this.
I can remove it as well.

Reply:
done